source/plugin/api-versions.rst

```diff
@@ -0,0 +1,74 @@
+============
+API-Versions
+============
+
+This page explains which API versions exist, and to which Minecraft version their implementations belong.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *8.0.0*     | TBA          | TBA            | * *SpongeForge (1.13.x)*                  |
+|             |              |                | * *SpongeVanilla (1.13.x)*                |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.1.0       | Sep 6, 2018  | TBA            | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.0.0       | Jan 1, 2018  | TBA            | * SpongeForge (1.12.2)                    |
```

Comment:
`TBA` -> `TBD`

Reply:
For all entries or only the 7.0.0 one?

Reply:
TBA, i.e. "To Be Announced" for the date when we cut off support seems appropriate; why use TBD instead? It's a bit vague, as it can mean "to be discussed" "to be done", "to be defined", "to be decided", "to be determined", "to be deleted", “to be denounced”.

Reply:
I was thinking TBD because it's something to be determined instead of something pending announcement.